### ISSUE TITLE
Catchable exception warnings

### ISSUE THREAD

**Issue by jfilby** [flyx/NimYAML#133]
When I write code that call's this package's load() function I get warnings:

```nim
/.../nexus/src/nexus/cmd/service/generate/modules/module_utils.nim(118, 9) template/generic instantiation of `load` from here
/home/jasonfi/.nimble/pkgs/yaml-1.1.0/yaml/serialization.nim(1383, 12) template/generic instantiation of `yAssert` from here
/home/jasonfi/.nimble/pkgs/yaml-1.1.0/yaml/private/internal.nim(42, 9) Warning: The bare except clause is deprecated; use `except CatchableError:` instead [BareExcept]
```

I tried wrapping the call in try/except CatchableError, but it didn't help. I think the problem is in the YAML module. This is quite a new warning. I'm using Nim v1.6.12.

**Comment by flyx** [comment]
This is already fixed but has not found its way in a release yet. Probably should do one.

**Comment by flyx** [comment]
Fixed in released NimYAML 2.0.0.
